LibGit2/Functions/Update-GitRepository.ps1

```diff
@@ -0,0 +1,78 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+function Update-GitRepository
+{
+    <#
+    .SYNOPSIS
+
+    Creates a detached head state, and updates the repo to the given target
+
+    .DESCRIPTION
+
+    The `Update-GitRepository` function updates the repo to the given target. Unless this target is a branch name, the repo will then be in a detached head state.

```

Comment:
Change "update the repo to the given target" to "updates a Git repository to a specific commit, i.e. it checks out a specific commit". Remove the next sentence about being in a detached head state.